src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -85,6 +100,25 @@ const LoggedInHome: FC = () => {
 	 * Effects
 	 */
 
+	// Scroll to ReadingRoomCardsWithSearch when search is present in query
+	useEffect(() => {
+		if (
+			!hasScrolledToSearch &&
+			query[SEARCH_QUERY_KEY] &&
+			!isLoadingActive &&
+			!isLoadingFuture &&
+			!isLoadingPending &&
+			!query[VISITOR_SPACE_SLUG_QUERY_KEY]
+		) {
+			searchRef.current?.offsetTop &&
+				// Small timeout so page is rendered before scrolling to search
+				setTimeout(() => {
+					window.scrollTo({ top: searchRef.current?.offsetTop, behavior: 'smooth' });
```

Comment:
Nice to have: maybe extend the existing util function: scrollToTop and rename it to:
```
export function scrollTo(yLocation: number): void {
	window.scrollTo({ top: yLocation, left: 0, behavior: 'smooth' });
}
```